src/tests/ras/utils/configXML/local_dimm_configuration.cc

```diff
@@ -62,7 +62,7 @@ int LocalDimmConfiguration::FillConfigFields(pugi::xml_node &&root) {
   }
 
   if (SetTestDir(root, test_dir_) != 0 ||
-      SetDimmCollections(std::move(root.child("dimmConfiguration"))) != 0) {
+      SetDimmCollections(root.child("dimmConfiguration")) != 0) {
```

Comment:
doesn't look like a part of the same commit. But it doesn't really change anything, so... ;)

Reply:
It is, this change was made due to clang werror